ml-pipeline/content.md

```diff
@@ -22,10 +22,20 @@ Run ```pip install mlflow[extras]==1.1.0 mflux-ai kedro==0.15.0 keras==2.2.4 ten
 In this tutorial we will create a complete machine learning pipeline using [Kedro](https://github.com/quantumblacklabs/kedro).
 We will create a pipline for the video classification problem.
 
+## Why Kedro?
+
+The main advantage of Kedro compared to competitors like Azure and AWS is that it is easy and fast to go from an experimentation stage to production.
+
+* It uses Python functions for defining the logic and steps in the pipeline.
+
+* It does not need a scheduler or database to set up a pipeline.
+
+* It starts with a project template, making it easy to set up and collaborate on a data science project.
```

Comment:
Another point:

* No vendor lock-in - it can be deployed e.g. on a laptop or in a cloud provider of choice